suspenders.gemspec

```diff
@@ -6,7 +6,6 @@ Gem::Specification.new do |s|
   s.required_ruby_version = ">= #{Suspenders::RUBY_VERSION}"
   s.required_rubygems_version = ">= 2.7.4"
   s.authors = ["thoughtbot"]
-  s.date = Date.today.strftime("%Y-%m-%d")
```

Comment:
How do we want to set the `date` for the gem? Can you say more about this line and the ramifications of it?

Reply:
@mike-burns Sure! The new version of `standardrb` (it is unversioned in our gemspec, so it always gets the latest version) linted against this line saying it is not necessary because it's already the default behavior. In other words, the date will be set to the current date by default.

Reply:
Aha! Nice.

I guess it's been like that for [nearly two decades](https://github.com/rubygems/rubygems/commit/3864a56b1054f0916c91ff8547653f16ce7cb52b)! Glad we found it, finally.